samza-sql/src/main/java/org/apache/samza/sql/interfaces/UdfMetadata.java

```diff
@@ -32,6 +32,7 @@
 public class UdfMetadata {
 
   private final String name;
+  private final String displayName;
```

Comment:
Might want to add a comment/example explaining the difference between name and displayName (if this isn't a well know samza concept)?